SeQuant/domain/mbpt/models/cc.cpp

```diff
@@ -148,4 +149,80 @@ std::vector<ExprPtr> CC::λ(bool screen, bool use_topology,
   return result;
 }
 
+std::vector<sequant::ExprPtr> CC::pert_t1() {
```

Comment:
to avoid confusion with singles T amplitudes, perhaps would be better (and future-proofier) to call this `t_pt(order)` and only check precondition that `order==1`?

Reply:
perhaps also makes sense to pass in perturbing operator rank (1 for 1-body pert, 2 for 2-body, etc.)